seed/challenges/03-back-end-development-certification/back-end-development-certificate.json

```diff
@@ -22,7 +22,7 @@
         [
           "http://i.imgur.com/2qn7tHp.jpg",
           "An image of the text \"Back End Development Certificate requirements\"",
-          "Let's confirm that you have completed our upper intermediate and advanced Bonfires, and our Basejumps. Click the button below to verify this.",
+          "Let's confirm that you have completed our upper intermediate algorithm scripting challenges, and our back end development projects. Click the button below to verify this.",
```

Comment:
They no longer need to complete the upper intermediate algorithm scripting challenges - please drop this entirely and just say "back end development projects"